app/templates/people/person/personal-information/index.hbs

```diff
@@ -0,0 +1,110 @@
+<div class="au-c-body-container au-c-body-container--scroll">
+  <div class="au-o-box au-o-flow au-o-flow--large">
+    <div class="au-o-flow au-o-flow--tiny">
+      <AuToolbar>
+        <AuToolbarGroup>
+          <AuHeading>
+            Persoonlijke gegevens
+          </AuHeading>
+        </AuToolbarGroup>
+        <AuToolbarGroup>
+          <AuLink
+            @route="people.person.personal-information.edit"
+            @skin="button-secondary"
+            @icon="pencil"
+            @iconAlignment="left"
+          >
+            Bewerk
+          </AuLink>
+        </AuToolbarGroup>
+      </AuToolbar>
+      <AuHeading @level="2" @skin="3">
+        {{@model.person.givenName}}
+        {{@model.person.familyName}}
+      </AuHeading>
+    </div>
+
+    <DataCard>
+      <:title>Persoonlijke gegevens</:title>
+      <:card as |Grid|>
+        <Grid as |Item|>
+          <Item>
+            <:label>Voornaam</:label>
+            <:content>{{@model.person.givenName}}</:content>
+          </Item>
+        </Grid>
+        <Grid as |Item|>
+          <Item>
+            <:label>Achternaam</:label>
+            <:content>{{@model.person.familyName}}</:content>
+          </Item>
+        </Grid>
+        {{#if @model.person.firstNameUsed}}
+          <Item>
```

Comment:
This needs to be nested under a grid component. I don't think it's valid at the moment.